.pipelines/templates/PackageBuild.yml

```
@@ -22,6 +22,10 @@ parameters:
     type: string
     default: ""
 
+  - name: extraPackageRetestList
```

Could we just call it `testRerunList`? I'm trying to be consistent and stick to duplicating the toolkit's names, so it's easier to understand what these parameters mean for people familiar with the toolkit arguments.
```suggestion
  - name: testRerunList
```

---

Sure, can do that